### PR TITLE
Bug 1139490: Fix hardcoded HTML/Python/JS paths

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by jezdez** [comment]
@openjck That's an intermediate step to replacing those paths prefixes with STATIC_URL at some point, right?

**Comment by openjck** [comment]
Definitely. The ultimate goal is to move everything into `kuma/static` and use `staticfiles_storage.url` to generate asset paths. Standardizing on `MEDIA_URL` in the meantime will help me find/update the paths in one go when we get to that point.

**Comment by jezdez** [comment]
Cool, that makes total sense. Thanks!

File: kuma/wiki/templates/wiki/includes/document_macros.html [hunk 1]
```diff
@@ -216,7 +216,7 @@
     {% set short_share_url_gplus = share_url_gplus | bitly_shorten %}
 
     {% set twitter_url = 'http://twitter.com/share'|urlparams(url=short_share_url_twitter, text=share_text) %}
-    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]='+short_share_url_facebook+'&p[images][0]=/media/redesign/img/favicon144.png&p[title]=' + share_text %}
+    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]='+short_share_url_facebook+'&p[images][0]=' + MEDIA_URL + 'img/favicon144.png&p[title]=' + share_text %}
```
Comment (openjck) [inline kuma/wiki/templates/wiki/includes/document_macros.html]: Note to self: I need to look into whether a full URL (like `https://developer.cdn.mozilla.net...`) would be valid here.

File: media/js/social.js [hunk 2]
```diff
@@ -19,7 +19,7 @@
             track('Twitter');
         }
         else if($this.hasClass('facebook')) {
-            open('http://facebook.com/sharer.php?s=100&p[url]=' + url + '&p[images][0]=/media/redesign/img/favicon144.png&p[title]=' + title, 'facebook-share', 'height=380,width=660,resizable=0,toolbar=0,menubar=0,status=0,location=0,scrollbars=0');
+            open('http://facebook.com/sharer.php?s=100&p[url]=' + url + '&p[images][0]=' + mdn.mediaPath + 'img/favicon144.png&p[title]=' + title, 'facebook-share', 'height=380,width=660,resizable=0,toolbar=0,menubar=0,status=0,location=0,scrollbars=0');
```
Comment (openjck) [inline media/js/social.js]: Note to self: I need to look into whether a full URL (like `https://developer.cdn.mozilla.net...`) would be valid here.

File: kuma/wiki/templates/wiki/includes/document_macros.html (hunk 1 above)
Comment (openjck) [inline kuma/wiki/templates/wiki/includes/document_macros.html]: [This article](http://ar.zu.my/how-to-really-customize-the-deprecated-facebook-sharer-dot-php/) implies that full URLs are valid. Regardless, [Facebook says](https://developers.facebook.com/x/bugs/357750474364812/) that arguments are ignored.

That seems to be consistent with the behavior I'm seeing. The following link specifies google.com as the URL and our favicon as the image, but the Google logo is used as the image anyway.

http://facebook.com/sharer.php?s=100&p[url]=https://google.com&p[images][0]=https://developer.cdn.mozilla.net/media/img/favicon144.png&p[title]=I%20learned%20about%20Template:test%20on%20MDN.

I'm gonna go ahead and remove the image argument. cc @stephaniehobson

File: media/js/social.js (hunk 2 above)
Comment (openjck) [inline media/js/social.js]: Same as [here](https://github.com/mozilla/kuma/pull/3406#discussion_r36879233).

File: kuma/wiki/templates/wiki/includes/document_macros.html [hunk 3]
```diff
@@ -216,7 +216,7 @@
     {% set short_share_url_gplus = share_url_gplus | bitly_shorten %}
 
     {% set twitter_url = 'http://twitter.com/share'|urlparams(url=short_share_url_twitter, text=share_text) %}
-    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]='+short_share_url_facebook+'&p[images][0]=/media/redesign/img/favicon144.png&p[title]=' + share_text %}
+    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]=' + short_share_url_facebook %}
```
Comment (openjck) [inline kuma/wiki/templates/wiki/includes/document_macros.html]: https://developers.facebook.com/x/bugs/357750474364812/

File: media/js/social.js [hunk 4]
```diff
@@ -19,7 +19,7 @@
             track('Twitter');
         }
         else if($this.hasClass('facebook')) {
-            open('http://facebook.com/sharer.php?s=100&p[url]=' + url + '&p[images][0]=/media/redesign/img/favicon144.png&p[title]=' + title, 'facebook-share', 'height=380,width=660,resizable=0,toolbar=0,menubar=0,status=0,location=0,scrollbars=0');
+            open('http://facebook.com/sharer.php?s=100&p[url]=' + url, 'facebook-share', 'height=380,width=660,resizable=0,toolbar=0,menubar=0,status=0,location=0,scrollbars=0');
```
Comment (openjck) [inline media/js/social.js]: https://developers.facebook.com/x/bugs/357750474364812/

**Comment by openjck** [comment]
Updated! This is all set. :smile:

**Comment by darkwing** [comment]
Merge conflict :(

**Comment by openjck** [comment]
Rebased! It was conflicted about the removed `calendar.js` file.

**Comment by darkwing** [comment]
Got a few errors when minifying:

https://gist.github.com/darkwing/ff548fd877ab49acc0f2

**Comment by darkwing** [comment]
Hold on, I think I may have accidentally done something to cause that error

**Comment by darkwing** [comment]
Sorry, there is just the one error:  https://gist.github.com/darkwing/4bf0d2dec6633bd23bda

**Comment by openjck** [comment]
Nice catch, updated.

**Comment by darkwing** [comment]
R+!  All checks work for me.

**Pull request merged**